breathecode/activity/tests/urls/tests_cohort.py

```diff
@@ -0,0 +1,83 @@
+"""
+Test /answer
+"""
+from django.utils import timezone
+from datetime import timedelta
+from unittest.mock import MagicMock, call, patch
+
+from django.urls.base import reverse_lazy
+from rest_framework import status
+
+from breathecode.services.google_cloud import Datastore
+
+from ..mixins import MediaTestCase
+
+DATASTORE_SEED = [
+    {
+        'academy_id': 0,
+        'cohort': None,
+        'created_at': (timezone.now() + timedelta(days=1)).isoformat() + 'Z',
+        'data': None,
+        'day': 13,
+        'email': 'konan@naruto.io',
+        'slug': 'breathecode_login',
+        'user_agent': 'bc/test',
+        'user_id': 1,
+    },
+]
+
+
+def datastore_fetch_mock(first_fetch=[]):
+    def fetch(**kwargs):
+
+        return first_fetch
+
+    return MagicMock(side_effect=fetch)
+
+
+class MediaTestSuite(MediaTestCase):
+    """
+    🔽🔽🔽 With data
+    """
+    @patch.object(Datastore, '__init__', new=lambda x: None)
+    @patch.object(Datastore,
+                  'fetch',
+                  new=datastore_fetch_mock(first_fetch=DATASTORE_SEED))
+    def test_get_activities(self):
```

Comment:
We need a test:

1. With more that 100 activities without passing paginations that just get 100 elements
2. With 20 elements, the `limit` could be 10, the `offset` could be 0, this returns the first 10 elements
3. With 20 elements, the `limit` could be 10, the `offset` could be 10, this returns the last 10 elements
4. With 20 elements, the `limit` could be 10, the `offset` could be 20, this returns zero elements

This test should copy and paste to `activity/me` and remember you should use a generator to generate a list with 10 or 100 elements of same dictionary